rpki-rtr-server/src/main/resources/application.properties

```diff
@@ -47,22 +47,25 @@ server.port=8081
 rtr.server.port=8323
 
 #
-# By default rpki-rtr-server will listen on localhost ONLY for RTR connections
+# You can specify the address the rpki-rtr-server listens on by setting the
+# rtr.server.address property. If no server.address is set rtr-server will listen
+# on all interfaces (on IPv4 and IPv6).
+# If you want to explicitly listen to all (IPv4) addresses use 0.0.0.0
 #
+# When you do we recommend that you limit access to this service in
```

Comment:
"When you do we recommend that you limit access to this service in your firewall"
When you do what?

Reply:
This is a verbatim copy of somewhere else in the file.

Proposal:
```suggestion
# When your rpki-rtr-server listens on all interfaces we recommend that you
# limit access to this service using a firewall.
```